src/VisualStudio/LiveShare/Impl/Client/Razor/CSharpLspContainedLanguageProvider.cs

```diff
@@ -50,6 +51,7 @@ public IVsContainedLanguage GetLanguage(string filePath, IVsTextBufferCoordinato
                 _razorProjectFactory.Workspace,
                 projectId,
                 project: null,
+                LanguageNames.CSharp,
```

Comment:
This probably needs to be the LSP language name StringConstants.CSharpLspLanguageName.